src/globals.cpp

```diff
@@ -287,6 +287,8 @@ CTweakRef<bool> enableDataSigVerifyTweak("consensus.enableDataSigVerify",
     "true if OP_DATASIGVERIFY is enabled.",
     &enableDataSigVerify);
 
+CTweak<unsigned int> numMsgHandlerThreads("net.msgHandlerThreads", "Number of message handler threads", 4);
+
```

Comment:
I don't think this makes sense as a tweak. You can't actually tweak it and make any more threads start or stop.   (Would be good also to default it to something like 50% of available threads, but that can be done later.)

Reply:
good point.  I want to dynamically add or expire threads up to this configured maximum, but this PR doesn't get there yet.